Sump/doc/spec.md

```diff
@@ -0,0 +1,31 @@
+Entity: Sump
```

Comment:
definitions here to the shcema.json according to the [contribution manual](https://bit.ly/contribution_manual) pages 8 and 9